compiler/x/amd64/codegen/TreeEvaluatorTable.hpp

```diff
@@ -613,7 +613,8 @@
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::ResolveCHK (J9)
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::ResolveAndNULLCHK (J9)
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::DIVCHK (J9)
-   TR::TreeEvaluator::OverflowCHKEvaluator,                // TR::OverflowCHK 
+   TR::TreeEvaluator::OverflowCHKEvaluator,                // TR::OverflowCHK
+   TR::TreeEvaluator::badILOpEvaluator,                    // TR::UnsignedOverflowCHK 
```

Comment:
is the idea that a subsequent commit will implement an unsigned overflow evaluator?

If that's the case, please open an issue now to implement the unsigned overflow evaluator. I'm also slightly concerned that we're building technical debt on other platforms by only implementing these things on x86 (only amd64 even). Perhaps we need an issue to implement the overflowchk evaluators on other platforms.